CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Documentation for rocRAND is available at
 
 ### Additions
 
+* GFX1034 Support
 * GFX1151 Support
```

Comment:
```suggestion
### Added

* Support for the gfx1034 and gfx1151 architectures
```

Reply:
@amd-mtrifuno: We're trying to convert changelogs to the new format. Can we use this heading and wording instead. I already have a PR for the previous changes at https://github.com/ROCm/rocRAND/pull/563, but maybe we can close that and use this instead to avoid conflicts.